Sources/TuistCore/Graph/Graph.swift

```diff
@@ -225,6 +225,13 @@ public class Graph: Encodable, Equatable {
             references = references.union(transitiveSystemLibraries)
         }
 
+        if targetNode.target.isAppClip {
```

Comment:
@pepibumur i don't know to test it as the AppClip.framework path refers to /Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk/System/Library/Frameworks/AppClip.framework." which it is complaining of missing